--- as2cfg.l.go	2019-12-29 21:57:13.000000000 +0800
+++ as2cfg.l.go.right	2019-12-29 21:57:05.000000000 +0800
@@ -252,10 +252,6 @@
 		goto yyrule1
 	case this.current == ':':
 		goto yystate25
-	case this.current == '>':
-		goto yystate24
-	case this.current >= '\x01' && this.current <= '\t' || this.current >= '\v' && this.current <= '9' || this.current >= ';' && this.current <= '=' || this.current >= '?' && this.current <= 'ÿ':
-		goto yystate23
 	}
 
 yystate25:
@@ -263,10 +259,6 @@
 	switch {
 	default:
 		goto yyrule1
-	case this.current == '>':
-		goto yystate24
-	case this.current >= '\x01' && this.current <= '\t' || this.current >= '\v' && this.current <= '=' || this.current >= '?' && this.current <= 'ÿ':
-		goto yystate23
 	}
 
 yystate26:
